src/robot/RobotApplication.java

```diff
@@ -0,0 +1,41 @@
+package robot;
+
+public class RobotApplication {
+    public static void main(String[] args) {
+        RobotConnectionManager robotConnectionManager = new RobotConnectionManager() {
+            @Override
+            public RobotConnection getConnection() {
+                return null;
+            }
+        };
+
+        moveRobot(robotConnectionManager, 1, 2);
+    }
+
+    public static void moveRobot(RobotConnectionManager robotConnectionManager, int toX, int toY) {
+        RobotConnection robotConnection = null;
```

Comment:
is this method work on stepik?

Reply:
I testing just moveRobot() method.

Reply:
send me screen in the slack how this method work in the stepik, please